nucleus/core/logging:target:test-events.log

```diff
@@ -0,0 +1,6 @@
+[#|2018-12-16T13:54:32.990+0100|INFO||javax.enterprise.test.logging.events|_ThreadID=1;_ThreadName=main;_TimeMillis=1544964872990;_LevelValue=800;|Test message for testLogEventListenerNotifications|#]
```

Comment:
Err...?

Reply:
eerrrhh.... where did THAT came from? :O